src/main/resources/assets/gregtech/lang/en_us.lang

```diff
@@ -4497,6 +4497,11 @@ gregtech.machine.muffle.off=Sound Muffling: Disabled
 gregtech.machine.perfect_oc=Does not lose energy efficiency when overclocked.
 gregtech.machine.parallel_limit=Can run up to §b%d§r§7 Recipes at once.
 
+gregtech.machine.copy_paste.tooltip=Shift Left-click with Data Stick to copy settings, right-click to apply
```

Comment:
Shift Right click now, not shift left click

Reply:
Japanese didn't seem to have existing lang for the data stick, but Russian and Chinese did, so those are wrong too. I see that there are 'update x lang for y version' PRs, so is it all right to let them take care of the other languages?

Reply:
Yes, that is fine